src/sql/azdata.proposed.d.ts

```diff
@@ -1333,6 +1337,33 @@ declare module 'azdata' {
 			 * Cost string for the node
 			 */
 			costDisplayString: string;
+			/**
+			 * Cost metrics for the node
+			 */
+			costMetrics: CostMetrics;
+		}
+
+		export interface CostMetrics {
+			/**
+			 * CPU Time taken by the node operation in milliseconds
+			 */
+			elapsedCpuTimeInMs: number | undefined;
```

Comment:
Can you please make this an array. This is not generic enough to be used in other databases.